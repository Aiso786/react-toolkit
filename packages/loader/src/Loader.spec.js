import React from 'react';
import renderer from 'react-test-renderer';
import Loader, { LoaderModes } from './index';

describe('<Loader>', () => {
  it('renders Loader correctly', () => {
    const tree = renderer
      .create(
        <Loader mode={LoaderModes.get} text="">
          <div>
            <h1>Title Child</h1>
            <span>Here your child component</span>
          </div>
        </Loader>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
