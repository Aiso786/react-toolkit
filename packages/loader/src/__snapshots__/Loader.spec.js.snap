// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader> renders Loader correctly 1`] = `
<div
  className={null}
  style={
    Object {
      "minHeight": "94px",
      "position": "relative",
    }
  }
>
  <div>
    <h1>
      Title Child
    </h1>
    <span>
      Here your child component
    </span>
  </div>
   
  <div
    className="af-loader af-loader-on"
    style={
      Object {
        "backgroundColor": "rgba(180, 180, 180, 0.75)",
        "height": "100%",
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      }
    }
  >
    <div
      className="af-spinner af-spinner--active"
    >
      <div
        className="af-spinner__animation"
      />
      <div
        className="af-spinner__caption"
      >
        Chargement en cours
      </div>
    </div>
  </div>
</div>
`;
